pkg/v1/config/clientconfig.go

```diff
@@ -46,6 +46,9 @@ const (
 	// AVI_MANAGEMENT_CLUSTER_CONTROL_PLANE_VIP_NETWORK_NAME and AVI_MANAGEMENT_CLUSTER_CONTROL_PLANE_VIP_NETWORK_CIDR
 	// when creating a cluster.
 	FeatureFlagManagementClusterNetworkSeparation = "features.management-cluster.network-separation-beta"
+	// ClusterClass feature flag determines whether to use clusterclass based management and workload
+	// cluster provisioning or legacy way of cluster provisioning
+	FeatureFlagClusterClass = "features.global.use-clusterclass-beta"
```

Comment:
Perhaps consider generalizing the feature flag to encompass the spectrum of work involving driving cluster lvm through packages (including clusterclass packages). 
Boringly how about features.global.package-based-lcm-beta

I don't feel too strongly about it. The other alternative is to stay as is, and expect other aspects within that spectrum to be gated via another ff and deal with complications of keeping their values consistent.

Reply:
I think it makes sense to have common flag. I will rename the ff.

Reply:
This name looks good.